test/e2etest/test-config.tfvars.tmpl

```diff
@@ -43,7 +43,6 @@ domain = "deplops.com"
 # Documentation can be found via:
 # https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html
 # https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Storage.html#USER_PIOPS
-bamboo_db_major_engine_version = "13"
```

Comment:
We should rely on the default value of database versions in e2e test rather than override it. In this case we can detect if the default version is not supported anymore in future.